testing/with-postgres-db.sh

```diff
@@ -1,11 +1,15 @@
 #!/bin/sh
 set -euo pipefail
 
+if hash greadlink &>/dev/null; then
+   readlink() { greadlink "$@"; }
+fi
+
 # set working directory to repo root
 cd "$(dirname "$(dirname "$(readlink -f "$0")")")"
 
 step() {
-  echo -e "\e[1;36m>>\e[0;36m $@...\e[0m"
+  echo -ie "\e[1;36m>>\e[0;36m $@...\e[0m"
```

Comment:
This breaks Linux because, over here, `echo`` does not know `-i`. Better use `printf` which is more portable.

Reply:
Fixed. This was a typo, I wanted to add the `e` flag to `sed -i ...`.